### PR TITLE
Make feature category an enum, and fix i18n for categories

### PR DESCRIPTION
**Pull request by magicus**
This should not go in before https://github.com/Wynntils/Artemis/pull/350.

### CODE REVIEW

**Comment by kristofbolyai** [comment]
Nice. I kinda needed this for setting screen anyways.

**Comment by kristofbolyai** [comment]
How can one get the category from the feature though?

**Comment by kristofbolyai** [comment]
> How can one get the category from the feature though?

Ah, I see, this is not fully implemented in Feature class yet.

**Comment by magicus** [comment]
@kristofbolyai I think this is okay now, I'll just verify that it actually works :)

**Comment by magicus** [comment]
Yeah, works fine

File: common/src/main/resources/assets/wynntils/lang/en_us.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 {
+  "core.wynntils.category.inventory": "Inventory",
```
Comment (kristofbolyai) [inline common/src/main/resources/assets/wynntils/lang/en_us.json:2]: woudln't feature.wynntils.category be better? As these are feature categories

Comment (kristofbolyai) [reply common/src/main/resources/assets/wynntils/lang/en_us.json:2]: Fine either way by me, though.

**Review by HighCrit — APPROVED** [review]
Only suggestion is bolyai's note but it's fine either way

File: common/src/main/resources/assets/wynntils/lang/en_us.json (hunk 1 above)
Comment (magicus) [reply common/src/main/resources/assets/wynntils/lang/en_us.json:2]: That was my first thought as well, but we have `feature.wynntils.<NAME>` as a pattern, where this is a reserved namespace for feature `<NAME>`. While we might never have a feature named CategoryFeature, I still felt it was mixing two different things.

**Pull request merged**